exercises/sam_jenkins/ex02_oo_basics/src/ShapeSorter.h

```diff
@@ -0,0 +1,17 @@
+#ifndef GUARD_ShapeSorter_h
+#define GUARD_ShapeSorter_h
+#include "Shapes.h"
+#include <vector>
+class ShapeSorter {
+public:
+	static void typePrint(std::string type,std::vector<Shapes*> shapes);
```

Comment:
Avoid static unless its absolutely necessary. Just make then normal methods and create an instance.

Reply:
Please pass `std::string` and `std::vector` by reference (or `const` reference) to avoid copying.